nixos/services/telegraf/psi_input.nix

```diff
@@ -1,12 +1,29 @@
-{ pkgs, ... }:
-{
-  flyingcircus.services.telegraf.inputs = {
-    exec = [{
-      commands = [ "${pkgs.fc.telegraf-collect-psi}/bin/collect_psi" ];
-      timeout = "10s";
-      data_format = "json";
-      json_name_key = "name";
-      tag_keys = ["period" "extent"];
-    }];
+{ pkgs, lib, config, ... }:
+let 
+  inherit (lib) types;
+  psiCgroupRegex = lib.concatStringsSep "|" config.flyingcircus.services.telegraf.psiCgroupRegex;
+in {
+  options.flyingcircus.services.telegraf.psiCgroupRegex = lib.mkOption {
+    type = types.listOf types.str;
+    default = [ ];
+  };
+  config = {
+    flyingcircus.services.telegraf.psiCgroupRegex = [ "^/system.slice$" "^/system.slice.*service$" "^/user.slice$" ];
```

Comment:
regex list should be empty by default, would produce too many metrics which our central statshost couldn't handle. Use the default regex as option example instead (and an option description would also be nice).